beacon_node/rest_api/src/validator.rs

```diff
@@ -78,6 +79,7 @@ pub fn get_validator_duties<T: BeaconChainTypes + 'static>(req: Request<Body>) -
             head_state
                 .get_beacon_proposer_index(slot, relative_epoch, &beacon_chain.spec)
```

Comment:
This call uses the committee cache for `relative_epoch`.

To ensure you have the minimal required caching for this, call:

```rust
beacon_state.build_committee_cache(relative_epoch, &beacon_chain.spec)
```

[Reference](https://github.com/sigp/lighthouse/blob/5d91d5948115b1f7efc7fd7e403dc5dbbcae715f/eth2/types/src/beacon_state.rs#L845-L860)